submodules/eks/variables.tf

```diff
@@ -134,6 +134,12 @@ variable "bastion_public_ip" {
   default     = null
 }
 
+variable "ssm_log_group_name" {
```

Comment:
You'll probably need to pass this through the top-level `variables.tf` to make it truly configurable